packages/components/organisms/f-loyalty/src/components/Header.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+    <div :class="$style['c-loyaltyHeader']">
+        <div :class="$style['c-loyaltyHeader-container']">
+            <bread-crumbs
+                :links="links"
+                :router-links="true" />
+            <media-element
+                :class="$style['c-loyaltyHeader-media']"
+                :text-size="fontSize"
+                :title="$t('header.title')"
+                :text="$t('header.text')"
+                :image-url="getImageBasedOnScreenSize"
+                image-align="center"
+                :flex="flexLayout"
+                :alt="$t('header.alt')"
+            />
+        </div>
+    </div>
+</template>
+
+<script>
+import BreadCrumbs from '@justeat/f-breadcrumbs';
+import MediaElement from '@justeat/f-media-element';
+import '@justeat/f-media-element/dist/f-media-element.css';
+import '@justeat/f-breadcrumbs/dist/f-breadcrumbs.css';
+import debounce from 'lodash.debounce';
+import { FONT_SIZE } from '@justeat/f-media-element/src/constants';
+
+const REQUIRED_WINDOW_WIDTH = 600;
```

Comment:
May be worth a comment as to what this is used for, as it's not obvious without going through the related code.

Reply:
Yes will note that its used for changing image based on window width.